src/login/login.component.scss

```diff
@@ -33,6 +33,18 @@ login {
   flex: 1;
 }
 
+.social-login-button {
+  color: white !important;
```

Comment:
Using `!important` seems a bit obtrusive. Did you play with specificity and is this the only way to accomplish this? Also, it's a bit weird that this scss file is not wrapped within the component element, that could greatly improve specificity.

Reply:
This is because `.md-raised` adds a more specific color rule which then overrides the custom rule. But perhaps a better approach is to use `.md-button.social-login-button` as selector? I've created a separate PR for that so let me know what you think there.

I agree the file should be wrapped in a component selector 👍